src/start/PasswordAuthView.js

```diff
@@ -50,6 +50,8 @@ class PasswordAuthView extends React.Component {
   };
 
   validateForm = () => {
+    Keyboard.dismiss();
```

Comment:
We can do this better.
We would want to rework the screen itself to fit in the space above the keyboard.
Can you rework, please?